### DISCUSSION TITLE
请问是否支持 根据环境变量打包生成不一样的文档？

### DISCUSSION THREAD

**Discussion by kk305cs** [vuepress-theme-hope/vuepress-theme-hope#2603] [Q&A] [answered]
想做这样一个功能：
根据不同的环境变量在页面上展示不同的结构
处理办法：
package.json 里设置了cross-env 怎样在自定义组件vue文件中使用process.env？2.0版本是不是不支持process.env?
![D8CA67238781EA8004CADFA2ACFA037A](https://user-images.githubusercontent.com/26682400/210124793-aba3c564-c58b-46f5-880c-03aae6d2bd69.jpg)

**Comment by Mister-Hope** [comment, marked as answer]
process.env 仅在 node 侧可用，如果需要在客户端代码使用请使用 define 定义相关变量到客户端。

**Reply by kk305cs** [reply]
好的 谢谢，我使用了cross-env 在 config.ts 中可以获取到 process.env，vue 页面里可以尝试你说办法
